packages/smooth_app/lib/cards/expandables/attribute_list_expandable.dart

```diff
@@ -38,8 +40,7 @@ class AttributeListExpandable extends StatelessWidget {
     final List<Widget> chips = <Widget>[];
     final List<Widget> cards = <Widget>[];
     for (final String attributeTag in attributeTags) {
-      Attribute attribute =
-          ProductPreferences.getAttribute(product, attributeTag);
+      Attribute attribute = ProductExtra.getAttribute(product, attributeTag);
```

Comment:
What we call attributeTag is the same thing as the attribute id right? If so I think we should rename all attributeTags / attributeTag to attributeIds / attributeId.